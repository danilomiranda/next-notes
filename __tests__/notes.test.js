/* eslint-env jest */

import React from 'react';
import Notes from '../pages/notes'
import renderer from 'react-test-renderer';

test('Snapshow of Notes', () => {
  const notes = renderer.create(
    <Notes items={[
      {
        "_id": 0,
        "title": "Stark",
        "date": "22/03/2018"
      },
      {
        "_id": 1,
        "title": "Lannister",
        "date": "01/04/2017"
      },
      {
        "_id": 2,
        "title": "Targaryen",
        "date": "22/04/2017"
      },
      {
        "_id": 3,
        "title": "Karstark",
        "date": "22/04/2017"
      },
      {
        "_id": 4,
        "title": "Bolton",
        "date": "02/07/2017"
      },
      {
        "_id": 5,
        "title": "Martell",
        "date": "02/10/2017"
      }
    ]}/>
  )

  let tree = notes.toJSON()
  expect(tree).toMatchSnapshot()
});