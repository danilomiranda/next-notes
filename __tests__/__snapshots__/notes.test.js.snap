// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshow of Notes 1`] = `
<div
  className="jsx-2030500139 main"
>
  <div
    className="jsx-2090446088"
  />
  <header
    className="jsx-1350539649 flex-container"
  >
    <span
      className="jsx-1350539649 title"
    >
      Notes
    </span>
  </header>
  <div
    className="jsx-2030500139 page"
  >
    <div
      className="jsx-1433830949 search"
    >
      <input
        className="jsx-1433830949"
        onChange={[Function]}
        placeholder="Pesquise por uma nota"
        type="text"
        value=""
      />
    </div>
    <div
      className="jsx-944670893"
    >
      <h3
        className="jsx-944670893"
      >
        Notas
      </h3>
      <div
        className="jsx-944670893 item"
      >
        <div
          className="jsx-944670893 note"
        >
          <div
            className="jsx-4283392587"
          >
            <div
              className="jsx-4283392587 title"
            >
              Stark
            </div>
            <div
              className="jsx-4283392587 date"
            >
              22/03/2018
            </div>
          </div>
        </div>
      </div>
      <div
        className="jsx-944670893 item"
      >
        <div
          className="jsx-944670893 note"
        >
          <div
            className="jsx-4283392587"
          >
            <div
              className="jsx-4283392587 title"
            >
              Lannister
            </div>
            <div
              className="jsx-4283392587 date"
            >
              01/04/2017
            </div>
          </div>
        </div>
      </div>
      <div
        className="jsx-944670893 item"
      >
        <div
          className="jsx-944670893 note"
        >
          <div
            className="jsx-4283392587"
          >
            <div
              className="jsx-4283392587 title"
            >
              Targaryen
            </div>
            <div
              className="jsx-4283392587 date"
            >
              22/04/2017
            </div>
          </div>
        </div>
      </div>
      <div
        className="jsx-944670893 item"
      >
        <div
          className="jsx-944670893 note"
        >
          <div
            className="jsx-4283392587"
          >
            <div
              className="jsx-4283392587 title"
            >
              Karstark
            </div>
            <div
              className="jsx-4283392587 date"
            >
              22/04/2017
            </div>
          </div>
        </div>
      </div>
      <div
        className="jsx-944670893 item"
      >
        <div
          className="jsx-944670893 note"
        >
          <div
            className="jsx-4283392587"
          >
            <div
              className="jsx-4283392587 title"
            >
              Bolton
            </div>
            <div
              className="jsx-4283392587 date"
            >
              02/07/2017
            </div>
          </div>
        </div>
      </div>
      <div
        className="jsx-944670893 item"
      >
        <div
          className="jsx-944670893 note"
        >
          <div
            className="jsx-4283392587"
          >
            <div
              className="jsx-4283392587 title"
            >
              Martell
            </div>
            <div
              className="jsx-4283392587 date"
            >
              02/10/2017
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
